src/components/message/message.twig

```diff
@@ -0,0 +1,18 @@
+{% set tagName = tagName | default('div') %}
+
+<{{tagName}} class="c-message{% if class %} {{class}}{% endif %}">
+  {% block content %}
+    <p>
+      {{label | default('Hi, I\'m a message!')}}
+    </p>
+  {% endblock %}
+  {% if includeClose %}
+    <button class="c-message__close" type="button">
+      {% include '../../assets/icons/close.svg.twig' with {
+          class: 'c-icon',
+          aria_hidden: true
+        } %}
+      <span class="u-hidden-visually">Close</span>
```

Comment:
Might we be able to push this a bit further? 🔍  

If I were to close my eyes using a screen-reader and tabbed onto this button, I'd hear "close". I might start wondering though, what am I closing? 🤔 

If I can see, I visually understand the context _around_ the close button. It is clear that the close button resides within the message/notification element. 

Perhaps we could add some extra context so that screen reader users can receive a more equivalent experience? What about something like "Close the notification"? 🙂